MOBILE_API.xml

```diff
@@ -550,7 +550,12 @@
         <element name="VEHICLEDATA_FUELLEVEL" until="7.0"/>
         <element name="VEHICLEDATA_FUELLEVEL_STATE" until="7.0"/>
         <element name="VEHICLEDATA_FUELCONSUMPTION" />
-        <element name="VEHICLEDATA_EXTERNTEMP" />
+        <element name="VEHICLEDATA_CLIMATEDATA" since="7.1"/>
+        <element name="VEHICLEDATA_EXTERNTEMP" until="7.1">
```

Comment:
`VEHICLEDATA_EXTERNTEMP` should be marked as `deprecated="true" since="7.1"`

Reply:
@LitvinenkoIra, can you please make this update?

Reply:
Done in bdbdaf4fc14742361f4eb0a858ed1fe79834c692